kubernetes/container-azm-ms-agentconfig.yaml

```diff
@@ -1,105 +1,105 @@
-kind: ConfigMap

```

Comment:
if its whitespace only, can you undo changes to this file?

Reply:
> if its whitespace only, can you undo changes to this file?

> sure, let me do it.

Reply:
> > if its whitespace only, can you undo changes to this file?
> 
> > sure, let me do it.

Done.